### ISSUE TITLE
Update container image to v3.2.0

### ISSUE THREAD

**Issue by Idesmi** [savoirfairelinux/opendht#724]
Version 3.2.0 was not published as a container image because of build failure.
I ask if it was possible to make it available, please.

**Comment by ghenry** [comment]
Still an issue? Thanks.

**Comment by Idesmi** [comment]
> Still an issue?

Yes. See <https://github.com/savoirfairelinux/opendht/pkgs/container/opendht%2Fopendht-alpine>

**Comment by aberaud** [comment]
It's now updated correctly. Thanks for the report.
